bytes/src/lib.rs

```diff
@@ -54,20 +55,24 @@ pub mod rc {
         /// Importantly, this is unavailable for as long as the struct exists, which may
         /// prevent shared access to ptr[0 .. len]. I'm not sure I understand Rust's rules
         /// enough to make a strong statement about this.
-        sequestered: Rc<B>,
+        sequestered: Rc<Box<Any>>,
```

Comment:
Why did you decide to switch to `Box<Any>`?

Reply:
Ah, this was because otherwise the `B` type pervades the rest of the code, and everything needs to be generic with respect to `B`, or written for one specific `B`, and it seemed likely that we might have more than one type (e.g. `Vec<u8>` for process-local stuff, and maybe something different (`Arc<Vec<u8>>`?) for networking stuff).

(on the road at the moment, will try to be more helpful soon!)